### DISCUSSION TITLE
Beta testing

### DISCUSSION THREAD

**Discussion by TomUgulini** [broofa/flightcard#9] [Support & Help] [answered]
I started playing with the app - nice job!
I wanted to create some fake flightcards just to learn and test the system, but didn't want to add junk to your database.  
How do you want to handle new user testing?  Perhaps a "test" launch event we could check in to?

**Comment by broofa** [comment, marked as answer]
Short answer: "_Have fun and don't worry about polluting the DB_".  I'm still in hack-n-slash mode with the code, and regularly reset the database.  If you corrupt something, it'll help me identify where I need to add more guard rails.

Longer answer: The `main` branch auto-deploys to https://flightcard.org , so you can play with it there without having to install it.  The caveat is that I haven't setup different DBs for dev and production - all versions of the app connect to the same database(!) right now.  So at any given time, the app may break because I've changed the schema, or am working on [the DB access rules](https://firebase.google.com/docs/database/security).

Like I said... hack-n-slash mode. 🤷 

Long-term answer:  I need to set up different configurations for dev, testing, and production.  See #10
